app/Repositories/Eloquents/PayloadHistoryRepository.php

```diff
@@ -16,4 +18,20 @@ public function find($id)
     {
         return $this->model->with('webhook')->find($id);
     }
+
+    public function getAllAndSearch($perPage, $searchParams)
+    {
+        $query = PayloadHistory::whereHas('webhook', function ($wh) {
+            $wh->where('user_id', Auth::id());
+        })->orderBy('payload_histories.created_at', 'desc');
+        if ($searchParams) {
+            $searchParams = $this->handleSearchParams(['webhook', 'status'], $searchParams);
+
+            return $query->search($searchParams, $perPage);
+        } else {
+            return $query->paginate($perPage);
+        }
+    }
+
+
 }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Classes.ClassDeclaration.CloseBraceAfterBody</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PayloadHistoryRepository.php</br>**Line**: 37</br>The closing brace for the class must go on the next line after the body